tools/objc/gcov_stub

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
```

Comment:
hrm based on the code in rules_apple, this might work inside google? https://github.com/bazelbuild/rules_apple/blob/0a1cbce9b3d169269313aba1113d2aa0a8c3c514/apple/internal/testing/apple_test_rule_support.bzl#L124-L128